components/prospects/ProfileActions.tsx

```diff
@@ -0,0 +1,81 @@
+import { grey } from '@mui/material/colors';
+import ConfirmModal from '../helper/ConfirmModal';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import { useState } from 'react'
+
+function ProfileActions({ id, name, options }) {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [modalOpen, setModalOpen] = useState(false);
+  const [selectedOption, setSelectedOption] = useState
+  <{ label: string; action: () => void }>(null);
+  
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  }
+
+  const handleOptionClick = (option: { label: string; action: () => void }) => {
+    setSelectedOption(option)
+    setModalOpen(true);
+    setAnchorEl(null);
+  }
+
+  const handleDialogClose = () => {
+    setModalOpen(false)
+  }
+
+  const onConfirmModel = () => {
```

Comment:
Let's be consistent through out the project and use functions instead of constants that stores functions